examples/hello.rs

```diff
@@ -58,7 +58,7 @@ impl Filesystem for HelloFS {
         }
     }
 
-    fn getattr(&mut self, _req: &Request, ino: u64, reply: ReplyAttr) {
+    fn getattr(&mut self, _req: &Request, ino: u64, _fd: Option<u64>, reply: ReplyAttr) {
```

Comment:
Can you name these `_fh` to match the rest of the code? Same comment for the other examples